app/views/admin/schools/induction_coordinators/new.html.erb

```diff
@@ -9,7 +10,7 @@
       <li>Address: <%= @school.full_address %></li>
     </ul>
 
-    <h2 class="govuk-heading-m">Who do you want to nominate?</h2>
+    <h1 class="govuk-heading-xl">Induction tutor for <%= @school.name %></h1>
```

Comment:
That will make two h1 headers, which will probably fail any accessibility checks (if we have one on this page)

Reply:
Yeah, I will get rid of the top one, and see what Shani says.